securedrop/journalist_app/admin.py

```diff
@@ -34,6 +36,11 @@ def manage_config():
             static_filepath = os.path.join(config.SECUREDROP_ROOT,
```

Comment:
We should save this to a temp file, then once we've successfully resized it move it to the old location. Right now if we throw an error thumbnailing it, we'll have a broken image.

It should go:

1. Save to tmp
2. Thumbnail that overwrites tmp
3. Move tmp to `static/i/logo.png` using `os.rename`

Reply:
Made this change